### PR TITLE
update data and lint

### PR DESCRIPTION
**Pull request by rymaju**

### CODE REVIEW

File: src/pages/jumpstart/JumpstartData.tsx [hunk 1]
```diff
@@ -1,50 +1,52 @@
 export default [
   {
-    title: 'Sunday Sept. 27th - Project Overview and Terminal Setup',
+    title: 'Online (Week of Sept. 27, out now) - Git Tutorial  Workshop',
     body:
-      "We'll be helping you set up the software you'll need to run the project and teaching you some basic terminal commands you'll need to be able to interact with your program.",
+      'A quick tour of Git, one of the most useful tools for development (not required for the course). It keeps track of your development history and lets you work collaboratively with others.',
```
Comment (connernilsen) [inline src/pages/jumpstart/JumpstartData.tsx]: Oh wait, sorry. Can you keep the first workshop too and have the git workshop be the second workshop (so add it as a new workshop)? So it should be Project Overview ... as workshop 1, Git Tutorial as workshop 2 (online), Frontend Basics as workshop 3...

**Review by connernilsen — APPROVED** [review]
That's almost everything, just one quick adjustment. Thanks for doing this!

File: src/pages/jumpstart/JumpstartData.tsx (hunk 1 above)
Comment (rymaju) [reply src/pages/jumpstart/JumpstartData.tsx]: Got it, no problem

**Review by connernilsen — APPROVED** [review]
Looks great! Merge when you're ready

**Pull request merged**